lib/jqlite.js

```diff
@@ -49,9 +49,16 @@
         if (typeof newElem == "string") {
           newElem = document.createElement(newElem);
         }
-        this.element.appendChild(newElem);
-        return new JQLite(newElem);
+        this.element[0].appendChild(newElem);
+        return new JQLite([newElem]);
       }
+
+      JQLite.prototype.remove = function() {
+        this.element.forEach(function(element) {
```

Comment:
I think we should err on the side of predictability. So non-terminal, and returning the removed element. It's easy enough just to call it's parent function from there.